src/Components/dashboard/Dashboard.js

```diff
@@ -0,0 +1,156 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { Avatar, Button } from "@mui/material";
+import ViewModuleIcon from "@mui/icons-material/ViewModule";
+import ViewStreamIcon from "@mui/icons-material/ViewStream";
+import PersonIcon from "@mui/icons-material/Person";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+
+import eventsInformation from "../../utils";
+import EventClickedComponent from "../eventClicked/EventClcked";
+import "./dashboardStyles.scss";
+
+const DashboardComponent = (props) => {
+  const { textAvatar, userName } = props;
+  //Hooks
+  const [viewEvents, setViewEvents] = useState(true);
+  const [eventClicked, setEventClicked] = useState("");
+  const [hiddenDashboard, setHiddenDashboard] = useState(false);
+
+  //functions
+  const changeViewEventsToColumn = () => {
```

Comment:
we dont need this function, you can call the `setViewEvents` inside an arrow function, check the comment below